docs/release_notes/v0.11.3.md

```diff
@@ -0,0 +1,22 @@
+  
+# Dapr 0.11.3
+
+## Fixes
+
+* Improve actor service health check relibility to avoid unexpected placement disconnection(https://github.com/dapr/dapr/pull/2292).
+* Do not track Actor ID for actor pending lock count metric(https://github.com/dapr/dapr/pull/2295)
+
+### Overview
+
+We have run load test for actor to evaluate the performance and relibility with the load test since the last milestone. The test discovered [the below issues](https://github.com/dapr/dapr/issues/2093) causing memory leak:
+
+1. [Fixed in this hotfix] Frequent reconnections to placement service when app http endpoint is unresponsive intermitently under high load scneario - Fixed, 0.11.3 hotfix
+2. [Fixed in this hotfix] Memory increases from http middleware when recording request metric - Fixed, 0.11.3 hotfix
+3. No RSS memory size reclaim in Kubernetes even after actors are deactivated - Expected, documentation improvement in 1.0 RC1
+4. Unnecessary memory allocation in actor invocation flow - WIP, 1.0 RC1
```

Comment:
WIP? Work in progress? Simply say, to be addressed in v1.0-rc1 release.

Reply:
removed all 1.0-rc1 plan in this release note.